### PR TITLE
Benchmarking indexing processes

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

**Comment by cbeer** [comment]
(I've combined two different issues in this pull request because they'll step on each others toes.. though i'm happy to be cajoled to do them serially)

File: .travis.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ notifications:
   email: false
 
 rvm:
-  - 2.2.2
+  - 2.2.3
```
Comment (ndushay) [inline .travis.yml]: This should include whatever rubies we run on dev, stage and prod boxes.

**Comment by ndushay** [comment]
Is this in preparation for writing to gdor logs?  I'm not sure how Benchmarkable works, but I assume it will write to the rails logs?

File: .travis.yml (hunk 1 above)
Comment (cbeer) [inline .travis.yml]: Agreed. I'm getting a puppet request together to update our machines to 2.2.3.

Comment (ndushay) [inline .travis.yml]: I would do both for now, then.

Comment (cbeer) [inline .travis.yml]: Fine. I'll remove it entirely, until we update the production boxes.

**Comment by cbeer** [comment]
I've updated this to write to the gdor logs.

**Pull request merged**